package.json

```diff
@@ -7,14 +7,17 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "prepare": "husky"
+    "prepare": "husky",
+    "db:test": "node src/lib/db/rxdb.js"
```

Comment:
does this script still apply? I thought that node can't run the db as it needs to run in the client?